.env.example

```diff
@@ -3,7 +3,7 @@ HOSTNAME=
 PORT=3000
 IP=0.0.0.0
 API_PATH=/api/v1/
-HISTORY_API_FALLBACK=
+HISTORY_API_FALLBACK=true
```

Comment:
Ok but please make sure that setting `false` actually turns it off, hint: [`yn`](https://npm.im/yn) (you would never ever guess the author 😛)

Reply:
Done. 

`false === false ? true : false` :trollface: